specification/cognitiveservices/OpenAI.Inference/tspconfig.yaml

```diff
@@ -20,3 +20,10 @@ options:
     partial-update: true
     enable-sync-stack: true
     generate-tests: false
+  "@azure-tools/typespec-ts":
+    emitter-output-dir: "{project-root}/generated"
+    generateMetadata: false
```

Comment:
@deyaaeldeen @MaryGao and progress on minimizing these properties we have to include in all the tspconfig files?

Reply:
@weshaggard This should be done in the latest typespec-ts emitter and we don't need to provide `generateMetadata` options.